study.md

```diff
@@ -38,7 +39,7 @@ manipulates it by sorting it and then reversing it in place (i.e., modifying the
 reference to the original).
 
 ```ruby
-# your answer here
+array.to_a.reverse
 ```
```

Comment:
try this one again with a method that will 'sort' the array. Don't forget to modify IN PLACE (see next answer)